README.md

```diff
@@ -17,17 +18,22 @@ Based on the [Native Terraform Module](https://www.terraform.io/docs/providers/a
 ## Example Usage
 ```hcl-terraform
 module "azure-media-services" {
-  source = "git@github.com:hmcts/cnp-module-azure-media-services"
-  location = "uksouth"
-  env = "prod"
-  product = "dm"
-  account_tier = "Premium"
+  source                   = "git@github.com:hmcts/cnp-module-azure-media-services"
+  location                 = "${var.location}"
+  env                      = "${var.env}"
+  common_tags              = "${var.common_tags}"
+  product                  = "${var.product}"
+  account_tier             = "Premium"
```

Comment:
not sure if the premium tier should be here where people can copy from?

either remove it since its optional or change to standard imo ?

```suggestion
  account_tier             = "Standard"
```

```suggestion
```